lib/services/sites.test.js

```diff
@@ -113,8 +113,8 @@ describe(_.startCase(filename), function () {
     it('throw error on missing host', function () {
       files.getFolders.returns(['a', 'b']);
       path.resolve.returns('z');
-      files.getYaml.onFirstCall().returns(getMockBadSite());
-      files.getYaml.onSecondCall().returns(getMockBadSite());
+      files.getYamlWithEnv.onFirstCall().returns(getMockBadSite());
+      files.getYamlWithEnv.onSecondCall().returns(getMockBadSite());
```

Comment:
This _looks_ like you just renamed a method, but the inputs and outputs are exactly the same.  Why is that?